src/harness.ts

```diff
@@ -47,6 +47,28 @@ const EVENT_HANDLERS = [
 ];
 
 /**
+ * An internal function which finds a VNode based on `key`
+ * @param target The VNode to search
+ * @param key The key to find
+ */
+function findVNodebyKey(target: VNode, key: string | object): VNode | undefined {
```

Comment:
I know this is an internal function, so I guess the same applies to `sendEvent` (from a consumer perspective)